apps/production/prod-rucio-daemons.yaml

```diff
@@ -11,6 +11,15 @@ conveyorThrottlerCount: 2
 judgeEvaluatorCount: 2
 tracerKronosCount: 3
 judgeCleanerCount: 2
+replicaRecovererCount: 1
+minosTemporaryExpirationCount: 1
+
+replicaRecoverer:
+  activeMode: "false"
```

Comment:
I doubt this string value will result in a boolean `false`. 
Can you please verify/confirm or change it to `false`?

Reply:
I know that this worked in integration: https://github.com/dmwm/rucio-flux/commit/ad0f3ea709018ae9238a803cf52bc5b1ec339435

Reply:
That's a boolean true - any non empty string would result in true

Reply:
Good point, I changed it. @eachristgr was wondering why loadtest requests are ending up in `P` state. The reason might be the same [1]

[1] https://github.com/dmwm/rucio-flux/blob/8af29295f9aa459fc8f0364e4da12bdb14b52e88/apps/production/loadtest-rucio-daemons.yaml#L36

Reply:
That is a string config, should not be the case. Also, I think that was resolved. Christos can confirm.

Can you also add a release tag please and squash all of it for a final review?

Reply:
Maybe this PR is not the place for this, but since I was asked I will reply and we can delete the comment later.

What we did with loadtest was to run conveyor-preparer daemons with '--ignore-availability'. However, the change was made manually in the running production deployment, which means that we must keep an eye on and manually re-do the change if needed. When [1] is merged, we can properly apply the change in flux [2].

Although, this change does not mean that the loadtest transfer requests will bypass the P state. Contrary, preparer will be able to process this requests even if the destination site is marked with 'availability_write=False'.

[1] https://github.com/rucio/helm-charts/pull/210
[2] https://github.com/dmwm/rucio-flux/blob/main/apps/base/rucio-daemons/cms-rucio-daemons.yaml#L98-L102

Reply:
It should be done now @dynamic-entropy , thank you.

Thanks for the explanation @eachristgr , it's clear now.